python/cudf_polars/tests/experimental/test_bind_to_gpu.py

```
@@ -71,187 +83,191 @@ def _reset_bind_state() -> None:
-            bind_to_gpu(policy)
-            bind_to_gpu(policy)
-            assert mock_bind.call_count == 1
+def _body_bind_called_once() -> None:
+    _reset_bind_state()
+    with patch(
+        "cudf_polars.experimental.rapidsmpf.frontend.hardware_binding.bind"
+    ) as mock_bind:
+        from cudf_polars.experimental.rapidsmpf.frontend.hardware_binding import (
+            HardwareBindingPolicy,
+            bind_to_gpu,
+        )
```

Ruff (0.15.12)

[warning] `import` should be at the top-level of a file

(PLC0415)